index.md

```diff
@@ -114,6 +114,12 @@ Die Links zu den Zoom-Räumen werden an die angemeldeten Personen per E-Mail ges
    <!---/ Umstieg auf Linux für Interessenten--->
    </td>
   </tr>
+ <tr>
+  <td></td>
```

Comment:
Why the empty cells?

Reply:
split the weeks, same below between first and second week

Reply:
I don't think you need empty cells for that. An empty row should be enough

Reply:
or add `style="margin-top: 10pt;"` to the row that should have more space above

Reply:
you need the empty cells, empty row only doesn't work

Reply:
i want the double lines to separate the weeks

Reply:
What about ` <td colspan="4"></td>`  ?